docs/faq/terminology.rst

```diff
@@ -0,0 +1,46 @@
+.. _Observation_Terminology:
+
+***********
+Terminology
+***********
+
+Scheduling
+==========
+
+Scheduling observations is a common task in astronomy, but there is a lack of
+formal terminology. Here we define the terminology as it will be used in astroplan
+for scheduling tasks. This is for consistency within the code and the documentation.
+
+The terms used are as follows:
+
+* **visit**: a period of time spent observing a single target
+* **observing block** (OB): the minimum schedulable block (tentative: a group of visits)
+* **priority**: number assigned to the **observing block** by the user that 
+  defines its precedence within the set of blocks to be scheduled. Should probably
+  also be on a 0->1 (0=no good, 1=good) scale, or rescaled within the scheduler
+* **rank**: like a **priority**, but defined by a time allocation committee (TAC) rather
```

Comment:
I wouldn't say this *has* to be by a TAC - maybe instead "defined by an external agency like a time allocation committee (TAC)".  Also, I think this is a bit problematic because it's assigned to an entire "proposal" which isn't really defined here.  I agree with @bmorris3 that "proposal" is a well-understood term in general, but in the context of this definition, it's not clear.

So perhaps this "rank" idea needs to be re-thought as something more generic roughly like "a priority that applies to a specific set of observing blocks, often set by an agent other than the observer".  Perhaps give the TAC & proposal as a specific *example* of this, but by phrasing it as relative to a set of OBs, it's clearer how it fits in with this framework.